screens/searching_request.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+
+export default class Searching extends React.Component {
```

Comment:
Make sure that your class names **always** match the file name - it should be `SearchingRequest`